app/services/court_report_due_sms_reminder_service.rb

```diff
@@ -1,35 +1,15 @@
-class CourtReportDueSmsReminderService
-  BASE_URL = Rails.application.credentials[:BASE_URL]
+class CourtReportDueSmsReminderService < SmsReminderService
   GENERATE_CASE_COURT_REPORT_LINK = "/case_court_reports"
 
   class << self
     def court_report_reminder(user, report_due_date)
-      return if !user[:receive_sms_notifications] || user[:phone_number].blank?
-
-      user_casa_org = user.casa_org
-      twilio_service = TwilioService.new(user_casa_org.twilio_api_key_sid, user_casa_org.twilio_api_key_secret, user_casa_org.twilio_account_sid)
-      sms_params = {
-        From: user_casa_org.twilio_phone_number,
-        Body: create_message(report_due_date),
-        To: user.phone_number
-      }
-      twilio_service.send_sms(sms_params)
+      send_reminder(user, create_message(report_due_date))
     end
 
     private
 
     def create_message(report_due_date)
-      "Your court report is due on #{report_due_date}. Generate a court report to complete & submit here: " + create_short_link
-    end
-
-    def create_short_link
-      if BASE_URL.blank?
-        raise "BASE_URL environment variable not defined"
-      end
-
-      short_url_service = ShortUrlService.new
-      short_url_service.create_short_url(BASE_URL + GENERATE_CASE_COURT_REPORT_LINK)
-      short_url_service.short_url
+      "Your court report is due on #{report_due_date}. Generate a court report to complete & submit here: " + create_short_link(GENERATE_CASE_COURT_REPORT_LINK)
```

Comment:
We should use the 'SmsBodyHelper' for text message bodies so they are easily accessible (stored in one place)

Reply:
Oh nice! Done!